migtests/tests/pg/unique-key-conflicts-test/validateAfterChanges

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env python3
```

Comment:
How about we also validate the contents of the unique columns details stored in metadb? I just want to be extra-careful here because it's a critical part. That would help us ensure that there are no false-positives in the unique columns list